packages/connect/src/connect.ts

```diff
@@ -77,6 +77,13 @@ function getNetwork(chainId?: number): Network | null {
       ensAddress: '0x98df287b6c145399aaa709692c8d308357bc085d',
     }
   }
+  if (chainId === 100) {
+    return {
+      chainId: 100,
+      name: 'xdai',
```

Comment:
Just as a note, I was thinking of starting to accept the chain as `{ chainId: number, name: string }` rather than only `chainId: number`, to handle staging environments.

Reply:
Nice, I like that.